test/e2e/framework.go

```diff
@@ -61,8 +61,15 @@ const (
 	agentContainerName   string = "antrea-agent"
 	antreaYML            string = "antrea.yml"
 	antreaIPSecYML       string = "antrea-ipsec.yml"
+	antreaCovYML         string = "antrea-coverage.yml"
+	antreaIPSecCovYML    string = "antrea-ipsec-coverage.yml"
 	defaultBridgeName    string = "br-int"
 
+	antreaControllerCovBinary string = "antrea"
+	antreaAgentCovBinary      string = "antrea"
+	antreaControllerCovFile   string = "antreacontroller.out"
+	antreaAgentCovFile        string = "antreaagent.out"
```

Comment:
please consider using a better name, e.g. `antrea-controller.cov.out
same for agent

Reply:
good suggestion, thanks, Antoni.